sql/4_2_2-to-4_3_1_upgrade.sql

```diff
@@ -609,4 +609,8 @@ INSERT INTO `list_options` (`list_id`,`option_id`,`title`,`notes`,`seq`) VALUES
 
 #EndIf
 
-
+#IfMissingColumn list_options list_id option_id PageValidation
```

Comment:
Should be instead:
``````
#IfNotRow2D list_options list_id lists option_id PageValidation